### PR TITLE
feat: new route to display files

### PR DESCRIPTION
**Pull request by ptbrowne**
Adds a new route `/show/:fileId` that shows a modal with the given file in an iframe. It is useful for the bank mobile application since the mobile intents do not work yet.

* When the user closes the modal, the displayed folder is the parent folder of the file.
* Some mobiles do not display the PDF in an iframe and instead try to download the file. Since we cannot know whether the iframe is displaying or downloading the file, a fallback message is shown 2s after the modal is shown "You can close this if you have downloaded the file".

On a mobile whose modal is not displaying the file : 

On a mobile displaying the file

This PR is a first step, we should have later on a proper viewer (maybe PDF.js) to display PDFs on all mobiles.

### CODE REVIEW

**Comment by y-lohse** [comment]
I also think it's more elegant if there's a single URL for files and folders. But I wouldn't say it's too important, cozy URLs are private and don't get shared, as long as they are explicit I think we're good. 
So I would say, if the impact on the code is small, merge the 2 routes, if it's not trivial, then it can stay like this.

File: src/drive/ducks/files/styles.styl [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.fileOpener__iframe
+    width 80vw
```
Comment (y-lohse) [inline src/drive/ducks/files/styles.styl]: We've had some really bad times with `vw` units, I don't knwo what @GoOz thinks about it but at this point I'd be more confident in using `%`.

**Comment by enguerran** [comment]
Well, if we want explicit routes, we shall rename `/files/:folderId` to `/folder/:folderId` and use `/file/:fileId`.

**Comment by ptbrowne** [comment]
I agree that having the two routes merged would be better. I was going for that at the beginning but by looking at the code, it was non trivial. I spoke yesterday to @goldoraf on this issue and he was fine having an additional route.

File: src/drive/ducks/files/styles.styl (hunk 1 above)
Comment (GoOz) [reply src/drive/ducks/files/styles.styl]: Actually it's `vh` the problem, `vw` not so much.

**Comment by ptbrowne** [comment]
OK, I'm making the change :)

**Pull request merged**